src/routes/core.js

```diff
@@ -110,32 +112,33 @@ router.get("/tagged/:tag.xml", async (req, res) => {
     params.last_updated = bookmarks[0].created_at;
   }
 
-  params.feedTitle = `${req.app.get('site_name')}: Bookmarks tagged '${req.params.tag}'`;
+  params.feedTitle = `${req.app.get('site_name')}: Bookmarks tagged '${tags.join(' and ')}'`;
   params.layout = false;
 
   res.type('application/atom+xml');
   return res.render("bookmarks-xml", params);
 });
 
-router.get("/tagged/:tag", async (req, res) => {
+router.get("/tagged/*", async (req, res) => {
+  const tags = req.params[0].split('/');
```

Comment:
is this safe? Say I'd put `; DROP TABLE bookmarks;` as the second tag. Would this be caught anywhere?

Reply:
It's a good question. The example you gave wouldn't work, but others might have. I modified the queries to use parameters. I've run `sqlmap` against the URL and it doesn't find any vulnerabilities.